src/Library/Host.cs

```diff
@@ -57,6 +60,11 @@ public void Run(string configuration, TimeSpan checkInterval)
                                     this.library = Host.StartNewLibrary(configuration, this.telemetryClient);
                                     this.isRunning = true;
                                 }
+
+                                if (this.ciLestener?.IsRunning != true)
+                                {
+                                    this.ciLestener = Host.StartNewWebServer(configuration);
+                                }
```

Comment:
Integration work on the other side is a while away, let's comment this out for now, doesn't make sense to have it run long before we need it.